build.gradle

```diff
@@ -20,28 +20,27 @@ repositories {
     }
 }
 
-configurations {
-    deployerJars
-}
-
+// java-library plugin defines 'api' configuration
+// 'api' configuration allows to expose dependencies with 'compile' scope in pom
+// 'implementation' configuration allows to expose dependencies with 'runtime' scope in pom
 dependencies {
     // web3j dependency
-    compile "org.web3j:core:4.7.0"
+    api "org.web3j:core:4.7.0"
     
     // apache commons.lang3
-    compile "org.apache.commons:commons-lang3:3.9"
+    api "org.apache.commons:commons-lang3:3.9"
 
     // multiaddresses (IPFS)
-    compile 'com.github.multiformats:java-multiaddr:1.3.1'
+    implementation 'com.github.multiformats:java-multiaddr:1.3.1'
 
     //jaxb required with Java 11
-    compile 'javax.xml.bind:jaxb-api:2.3.0'
+    implementation 'javax.xml.bind:jaxb-api:2.3.0'
 
     //args utils
-    compile 'org.apache.maven.shared:maven-shared-utils:3.2.1'
+    implementation 'org.apache.maven.shared:maven-shared-utils:3.2.1'
 
     // zip
-    compile 'net.lingala.zip4j:zip4j:2.3.1'
+    implementation 'net.lingala.zip4j:zip4j:2.3.1'
```

Comment:
Do you confirm we don't need it in worker?

Reply:
`grep -r 'zip4' src` is empty in iexec-worker.
I only see calls to com.iexec.common.utils.FileHelper and com.iexec.common.worker.result.ResultUtils.
We should be clean.

Reply:
Ok :+1: